Makefile

```diff
@@ -490,7 +490,7 @@ uki-certs: talosctl ## Generate test certificates for SecureBoot/PCR Signing
 .PHONY: cache-create
 cache-create: installer imager ## Generate image cache.
 	@docker run --entrypoint /usr/local/bin/e2e.test registry.k8s.io/conformance:$(KUBECTL_VERSION) --list-images | \
-		$(TALOSCTL_EXECUTABLE) images integration --installer-tag=$(IMAGE_TAG) --registry-and-user=$(REGISTRY_AND_USERNAME) | \
+		$(TALOSCTL_EXECUTABLE) images integration --installer-tag=$(IMAGE_TAG)-amd64-secureboot --registry-and-user=$(REGISTRY_AND_USERNAME) | \
```

Comment:
I guess unrelated to this, but to make the default secureboot installer?

Reply:
it is in fact related, the change you did to make ISO sd-boot broke image-cache, as it needs to cache a different installer image

Reply:
so I think we should figure out a plan vs. the `installer` and start using UKIfied installer everywhere now, but it doesn't have to be the this PR